spec/models/ems_refresh_spec.rb

```diff
@@ -280,4 +280,16 @@ def assert_queue_item(expected_targets)
       expect(MiqQueue.count).to eq(1)
     end
   end
+
+  describe ".create_refresh_task" do
+    it "create refresh task and trancates task name to 255 symbols" do
+      ems = double("Ems")
+      ems_name = "SomeTestEmsName"
+      allow(ems).to receive(:name).and_return(ems_name)
+      described_class.send(:create_refresh_task, ems, "a" * 1000)
+      task_name = MiqTask.last.name
```

Comment:
`EmsRefresh.create_refresh_task` return the task so you don't need to find it, just do `task = described_class.send(:create_refresh_task...`

Reply:
👍